dashboard/client/src/common/ProfilingLink.tsx

```diff
@@ -96,3 +142,127 @@ export const CpuProfilingLink = ({
     </Link>
   );
 };
+
+export const ProfilerButton = ({
+  profilerUrl,
+  type = "",
+}: MemoryProfilingButtonProps) => {
+  const [duration, setDuration] = useState<number | null>(5);
+  const [leaks, setLeaks] = useState<boolean>(true);
+  const [open, setOpen] = useState<boolean>(false);
```

Comment:
```suggestion
  const [leaks, setLeaks] = useState(true);
  const [open, setOpen] = useState(false);
```

nit: typescript can infer the type.